### ISSUE TITLE
Mapper type error

### ISSUE THREAD

**Issue by pierzen** [mvexel/meetyourmappers#9]
Looking for relation = 389536, some experienced mappers are listed as new mappers.

First edit seems to correspond to the first date the contributor contributed in this area. This makes confusion with contributor OSM total experience.  

I think thatOSM experience would be more relevant.

**Comment by mvexel** [comment]
You're talking about yourself here I presume? :)

<img width="1321" alt="screen shot 2018-08-16 at 9 10 44 am" src="https://user-images.githubusercontent.com/120452/44217261-4acb2a00-a134-11e8-9843-621ccb514e0e.png">

It looks like you are a new mapper **to this area**, I think that is a valid claim. The application only considers the local area you loaded. It doesn't make any claim as to overall experience. If you can suggest a way to make this clearer, let me know!

**Comment by pierzen** [comment]
Mapper type could be two fold and clarify, using **new in the area** instead of **new mapper**
(new in the area  / OSM experience  )
OSM experience could use categories like Neis or symbols expressing nb of years like  ¬¬¬¬

**Comment by pierzen** [comment]
Happy new year Martijn,

I have forked your p3-changesetMD https://github.com/mvexel/ChangesetMDand started to add revisions in the tree dopartial-log https://github.com/pierzen/ChangesetMD/tree/dopartial-log
I tried to enhance with Db Schema, more flexibility and logging infos.

- Db Schema - Added schema parameter and revised PostgreSQl instructions

- Your Batch-Rows routine (with capacity parameter) commits after each sequence. I revised it to commit to take account of the capacity throttle parameter (I renamed  bulkrows).

- Added doPartialReplication function(FromSeq, ToSeq) to let Custom Replications. This could be used for various projects to extract data for a few days that correpond to their event
- added FromSeq and ToSeq parameters -

- Logging - I added a logging file and integrated your timeCost function.
test subdirectory contains two powershell scripts to test. It automatically create Database and schema if non-existant. That make it easy to test.

Are you in contact with Toby Murray? Since he did not answer your pull request since 2019, I wonder if he still maintain the project.
I will also be pleased that you comment this revision before I go further with this. I dont know if there were any discussions about revisions in the last few years except I see that Pnorman asked to integrate db schemas.
 
Pierre 
 

    Le lundi 27 août 2018 10 h 45 min 14 s UTC−4, Martijn van Exel <notifications@github.com> a écrit :  
 
 
Closed #9 via 3287e37.
